dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json

```diff
@@ -171,118 +171,601 @@
+            },
+            {
+              "value": "support_student_understan",
+              "text": "support student understanding and use of of CS vocabulary."
+            },
+            {
+              "value": "support_students_in_becom",
+              "text": "support students in becoming informed decision makers about the impact of technology."
+            }
+          ],
+          "columns": [
+            {
+              "value": "0",
+              "text": "N/A"
+            },
+            {
+              "value": "1",
+              "text": "1"
+            },
+            {
+              "value": "2",
+              "text": "2"
+            },
+            {
+              "value": "3",
+              "text": "3"
+            },
+            {
+              "value": "4",
+              "text": "4"
+            },
+            {
+              "value": "5",
+              "text": "5"
+            },
+            {
+              "value": "6",
+              "text": "6"
+            },
+            {
+              "value": "7",
+              "text": "7"
+            }
+          ],
+          "visible": true
+        },
       {
-       "value": "6",
-       "text": "Agree"
+       "type": "matrix",
+       "name": "CSP_teaching_AP",
+       "title": "Please rate the extent to which you agree or disagree with the following statements about teaching an AP Computer Science course.",
+       "columns": [
+        {
+         "value": "0",
+         "text": "N/A"
+        },
+        {
+         "value": "1",
+         "text": "Strongly Disagree"
+        },
+        {
+         "value": "2",
+         "text": "Disagree"
+        },
+        {
+         "value": "3",
+         "text": "Slightly Disagree"
+        },
+        {
+         "value": "4",
+         "text": "Neutral"
+        },
+        {
+         "value": "5",
+         "text": "Slightly Agree"
+        },
+        {
+         "value": "6",
+         "text": "Agree"
+        },
+        {
+         "value": "7",
+         "text": "Strongly Agree"
+        }
+       ],
+       "rows": [
+        {
+         "value": "I_have_the_skills_and_kno",
+         "text": "I have the skills and knowledge that I need to teach this course as an AP course."
+        },
+        {
+         "value": "I’m_confident_my_students",
+         "text": "I’m confident my students can be successful on the AP CS Principles exam."
+        },
+        {
+         "value": "I’m_going_to_need_a_lot_o",
+         "text": "I’m going to need a lot of extra support preparing students for the AP exam and performance tasks."
+        },
+        {
+         "value": "This_workshop_increased_m",
+         "text": "This workshop increased my confidence that I can teach CS Principles as an AP course."
+        }
+       ]
       },
       {
-       "value": "7",
-       "text": "Strongly Agree"
+       "type": "comment",
+       "name": "question7",
```

Comment:
will want to change this name if you end up needing this question

Reply:
fixed.